### ISSUE TITLE
Miniposts Template uses page object, instead of minipost object to parse date

### ISSUE THREAD

**Issue by pquerner** [pmoreno-rodriguez/grav-theme-future2021#43]
See here:

https://github.com/pmoreno-rodriguez/grav-theme-future2021/blob/develop/templates/partials/miniposts.html.twig#L35-L39

```
{% if config.plugins["translate-date"].enabled %}
            <time class="published" itemprop="datePublished" datetime="{{ page.date|td(null, "Y-m-d") }}">{{ page.date|td(null, "M j, Y") }}</time>
          {% elseif config.plugins["twig-extensions"].enabled %}
            <time class="published" itemprop="datePublished" datetime="{{ page.date|localizeddate('medium', 'none', lang) }}">{{ page.date|localizeddate('medium', 'none', lang) }}</time>
          {% else %}
            <time class="published" itemprop="datePublished" datetime="{{ page.date|date("Y-m-d") }}">{{ page.date|date("M j, Y") }}</time>
          {% endif %}
```

instead of `page` should simply use `p` from

```
{% for p in page.evaluate({'@taxonomy.category':miniposts_category}).order('header.order', 'asc').slice(0,miniposts_number) %}
```

https://github.com/pmoreno-rodriguez/grav-theme-future2021/blob/develop/templates/partials/miniposts.html.twig#L11

**Comment by pmoreno-rodriguez** [comment]
Thanks @pquerner for your comment.
I'll fix this soon, for next version.

**Comment by pmoreno-rodriguez** [comment]
Fixed in 1.0.7 release.
